websites/R/Radiko/dist/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "Munn",
+    "id": "379616881272946688"
+  },
+  "contributors": [
+    {
+      "name": "Krauchelli",
+      "id": "229334514084741120"
+    }
+  ],
+  "service": "Radiko",
+  "description": {
+    "en": "Radiko is Japanese website that provide radio livestream in real time.",
```

Comment:
`Radiko is Japanese website that provides radio livestreaming in real time.` makes a little more sense.

Reply:
Hmm, i think so.